<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"/><title>Kinetic modeling predicts a stimulatory role for ribosome collisions at elongation stall sites in bacteria</title><meta name="viewport" content="width=device-width, initial-scale=1.0"><link rel="alternate" type="application/rss+xml" title="Bedford Lab Research" href="http://bedford.io/feed.xml"><link href="/css/style.css" rel="stylesheet" media="screen"><link rel="shortcut icon" href="/images/favicon.png"/> <script>!function(e,a,t,n,c,o,s){e.GoogleAnalyticsObject=c,e[c]=e[c]||function(){(e[c].q=e[c].q||[]).push(arguments)},e[c].l=1*new Date,o=a.createElement(t),s=a.getElementsByTagName(t)[0],o.async=1,o.src=n,s.parentNode.insertBefore(o,s)}(window,document,"script","//www.google-analytics.com/analytics.js","ga"),ga("create","UA-62729400-1","auto"),ga("send","pageview")</script><script src="/js/jquery-1.11.3.min.js"></script><script src="/js/collapse.js"></script></head><body><nav class="navbar navbar-default navbar-fixed-top" role="navigation"><div class="container"><div class="navbar-header"><div class="navbar-brand"> <a href="/"><img class="navbar-brand-logo" src="/images/logo.png"/></a></div><button type="button" class="navbar-toggle navbar-right" data-toggle="collapse" data-target="#navbar-menu-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button><div class="navbar-brand-text"><a href="/" class="off">Subramaniam Lab @ Fred Hutch</a></div></div><div class="collapse navbar-collapse" id="navbar-menu-collapse"><ul class="nav navbar-nav navbar-right text-right"><li> <a href="/research/">Research</a></li><li class="active"> <a href="/papers/">Papers</a></li><li> <a href="/members/">Team</a></li><li> <a href="/teaching/">Teaching</a></li><li><a href="https://rasilab.github.io/docs">Docs</a></li><li> <a href="/contact/">Contact</a></li><li> <a href="/join/">Join</a></li></ul></div></div> </nav><div class="container"><div class="bigspacer"></div><div class="row"><div class="col-md-12"><div class="media"><div class="media-body titlebox"><div class="smallhead text-bold"> Kinetic modeling predicts a stimulatory role for ribosome collisions at elongation stall sites in bacteria</div><p><h5> Ferrin MF, Subramaniam AR <sup><i class="small fa fa-envelope"></i></sup>. eLife 6:e23629, 2017</h5></div></div></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-3"> <img class="media-object biglogo" src="/images/papers/ferrin-elife-2017.png"></div><div class="col-md-5"><div class="post"><h4>Abstract</h4><p>Ribosome stalling on mRNAs can decrease protein expression. To decipher ribosome kinetics at stall sites, we induced ribosome stalling at specific codons by starving the bacterium <em>Escherichia coli</em> for the cognate amino acid. We measured protein synthesis rates from a reporter library of over 100 variants that encoded systematic perturbations of translation initiation rate, the number of stall sites, and the distance between stall sites. Our measurements are quantitatively inconsistent with two widely-used kinetic models for stalled ribosomes: ribosome traffic jams that block initiation, and abortive (premature) termination of stalled ribosomes. Rather, our measurements support a model in which collision with a trailing ribosome causes abortive termination of the stalled ribosome. In our computational analysis, ribosome collisions selectively stimulate abortive termination without fine-tuning of kinetic rate parameters at ribosome stall sites. We propose that ribosome collisions serve as a robust timer for translational quality control pathways to recognize stalled ribosomes.</p></div></div><div class="col-md-offset-1 col-md-3"><div class="bigspacer"></div><div class="glyphbox note"><div class="smallhead"> PDF</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-file-text-o fa-fw"></i> <a class="off" href="/pdfs/papers/ferrin-elife-2017.pdf">ferrin-elife-2017.pdf</a></div><div class="bigspacer"></div><div class="smallhead"> DOI</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-external-link fa-fw"></i> <a class="off" href="http://dx.doi.org/10.7554/eLife.23629">10.7554/eLife.23629</a></div><div class="bigspacer"></div><div class="smallhead"> Source code</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-github fa-fw"></i> <a class="off" href="https://github.com/rasilab/ferrin_elife_2017">ferrin_elife_2017</a></div><div class="bigspacer"></div></div></div></div></div><div class="hugespacer"></div><div class="hugespacer"></div><div class="hugespacer"></div><div class="hugespacer"></div> <nav class="navbar navbar-static-bottom" role="navigation"><div class="navbar-brand-centered footer">© Subramaniam Lab.<br/>All rights reserved. 2015–22.<br/>Design based on <a href="http://bedford.io" class="off">bedford.io</a></div> </nav> <script async  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script> <script async src="/js/mathjax.config.js"></script></body></html>