<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"/><title>Massively parallel identification of sequence motifs triggering ribosome-associated mRNA quality control</title><meta name="viewport" content="width=device-width, initial-scale=1.0"><link rel="alternate" type="application/rss+xml" title="Bedford Lab Research" href="http://bedford.io/feed.xml"><link href="/css/style.css" rel="stylesheet" media="screen"><link rel="shortcut icon" href="/images/favicon.png"/> <script>!function(e,a,t,n,c,o,s){e.GoogleAnalyticsObject=c,e[c]=e[c]||function(){(e[c].q=e[c].q||[]).push(arguments)},e[c].l=1*new Date,o=a.createElement(t),s=a.getElementsByTagName(t)[0],o.async=1,o.src=n,s.parentNode.insertBefore(o,s)}(window,document,"script","//www.google-analytics.com/analytics.js","ga"),ga("create","UA-62729400-1","auto"),ga("send","pageview")</script><script src="/js/jquery-1.11.3.min.js"></script><script src="/js/collapse.js"></script></head><body><nav class="navbar navbar-default navbar-fixed-top" role="navigation"><div class="container"><div class="navbar-header"><div class="navbar-brand"> <a href="/"><img class="navbar-brand-logo" src="/images/logo.png"/></a></div><button type="button" class="navbar-toggle navbar-right" data-toggle="collapse" data-target="#navbar-menu-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button><div class="navbar-brand-text"><a href="/" class="off">Subramaniam Lab @ Fred Hutch</a></div></div><div class="collapse navbar-collapse" id="navbar-menu-collapse"><ul class="nav navbar-nav navbar-right text-right"><li> <a href="/research/">Research</a></li><li class="active"> <a href="/papers/">Papers</a></li><li> <a href="/members/">Team</a></li><li> <a href="/teaching/">Teaching</a></li><li><a href="https://rasilab.github.io/docs">Docs</a></li><li> <a href="/contact/">Contact</a></li><li> <a href="/join/">Join</a></li></ul></div></div> </nav><div class="container"><div class="bigspacer"></div><div class="row"><div class="col-md-12"><div class="media"><div class="media-body titlebox"><div class="smallhead text-bold"> Massively parallel identification of sequence motifs triggering ribosome-associated mRNA quality control</div><p><h5> Chen KY, Park H, Subramaniam AR <sup><i class="small fa fa-envelope"></i></sup> bioRxiv 2023.09.27.559793, 2023</h5></div></div></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-3"> <img class="media-object biglogo" src="/images/papers/chen-biorxiv-2023.png"></div><div class="col-md-5"><div class="post"><h4>Abstract</h4><p>Decay of mRNAs can be triggered by ribosome slowdown at stretches of rare codons or positively charged amino acids. However, the full diversity of sequences that trigger co-translational mRNA decay is poorly understood. To comprehensively identify sequence motifs that trigger mRNA decay, we use a massively parallel reporter assay to measure the effect of all possible combinations of codon pairs on mRNA levels in <em>S. cerevisiae</em>. In addition to known mRNA-destabilizing sequences, we identify several dipeptide repeats whose translation reduces mRNA levels. These include combinations of positively charged and bulky residues, as well as proline-glycine and proline-aspartate dipeptide repeats. Genetic deletion of the ribosome collision sensor Hel2 rescues the mRNA effects of these motifs, suggesting that they trigger ribosome slowdown and activate the ribosome-associated quality control (RQC) pathway. Deep mutational scanning of an mRNA-destabilizing dipeptide repeat reveals a complex interplay between the charge, bulkiness, and location of amino acid residues in conferring mRNA instability. Finally, we show that the mRNA effects of codon pairs are predictive of the effects of endogenous sequences. Our work highlights the complexity of sequence motifs driving co-translational mRNA decay in eukaryotes, and presents a high throughput approach to dissect their requirements at the codon level.</p></div></div><div class="col-md-offset-1 col-md-3"><div class="bigspacer"></div><div class="glyphbox note"><div class="smallhead"> PDF</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-file-text-o fa-fw"></i> <a class="off" href="/pdfs/papers/chen-biorxiv-2023.pdf">chen-biorxiv-2023.pdf</a></div><div class="bigspacer"></div><div class="smallhead"> DOI</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-external-link fa-fw"></i> <a class="off" href="http://dx.doi.org/10.1101/2023.09.27.559793">10.1101/2023.09.27.559793</a></div><div class="bigspacer"></div><div class="smallhead"> Source code</div><div class="pad-left note"><div class="smallspacer"></div> <i class="fa fa-github fa-fw"></i> <a class="off" href="https://github.com/rasilab/chen_2023">chen_2023</a></div><div class="bigspacer"></div></div></div></div></div><div class="hugespacer"></div><div class="hugespacer"></div><div class="hugespacer"></div><div class="hugespacer"></div> <nav class="navbar navbar-static-bottom" role="navigation"><div class="navbar-brand-centered footer">© Subramaniam Lab.<br/>All rights reserved. 2015–22.<br/>Design based on <a href="http://bedford.io" class="off">bedford.io</a></div> </nav> <script async  src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script> <script async src="/js/mathjax.config.js"></script></body></html>